build/configs/artik05x/artik05x_download.sh

```diff
@@ -256,7 +273,7 @@ while test $# -gt 0; do
 		--verify)
 			VERIFY=verify
 			;;
-		ALL|OS|APPS|ROM|BL1|BL2|SSSFW|WLANFW|OTA|MICOM|WIFI|all|os|apps|rom|bl1|bl2|sssfw|wlanfw|ota|micom|wifi)
+		ALL|OS|APPS|ROM|BL1|BL2|SSSFW|WLANFW|OTA|MICOM|WIFI|all|os|apps|rom|bl1|bl2|sssfw|wlanfw|ota|micom|wifi|user)
```

Comment:
Don't we need to add |USERFS| between |WIFI| and |all|??

Reply:
Yes, USER needs to be added as well.